src/main/java/org/gitlab4j/api/models/DiffRef.java

```diff
@@ -0,0 +1,34 @@
+package org.gitlab4j.api.models;
+
+public class DiffRef {
+    private String base_sha;
+    private String head_sha;
+    private String start_sha;
+
```

Comment:
This library utilizes camel-case for variable names, so these should be:
```
    private String baseSha;
    private String headSha;
    private String startSha;
```

As should the getters and setters for same.